zap/src/main/java/org/zaproxy/zap/scan/filters/ScanFilter.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2019 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.zap.scan.filters;
```

Comment:
For now ascan package might be more appropriate until we discuss/decide if we want a common interface (and which) for all filters.

Reply:
ok will do that changes. but i think even in future if we come up with some other interface design then we can enhance this as we need this functionality in any case for all other scanner types.

Reply:
I was referring specifically to the `ScanFilter` interface, other generic filter classes can live here.

Reply:
oh ok. Makes sense.

Reply:
Sure, they would have to be adapted.